src/bank.cpp

```diff
@@ -69,4 +69,24 @@ extern "C" int openmc_fission_bank(void** ptr, int64_t* n)
   }
 }
 
+//==============================================================================
+// CPP API
+//==============================================================================
+std::vector<Particle::Bank>& openmc_source_bank_cpp()
```

Comment:
We could just call this `source_bank()`. It's in the `openmc` namespace already so no need to prefix it with `openmc_`. The suffix also is not very meaningful.

Reply:
My only issue witht his is shadowing the simulation::source_bank, but I guess its retrieving it. I fixed it like you requested, but let me know if you want me to call the cpp side get_source_bank (Pythong could still get it like openmc.core.source_bank()